herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -341,6 +341,84 @@ public TranslatedQuery translate(String defaultTableSpace, String query, List<Ob
         }
     }
 
+    private TranslatedQuery calculateShowCreateTable(String query, List<Object> parameters) {
+        String items[] = { "SHOW","CREATE", "TABLE"};
+        if(Arrays.stream(items).allMatch(query::contains)) {
+            query = query.substring(Arrays.stream(items).collect(Collectors.joining(" ")).length()).trim();
+            String tableSpace = "herd";
```

Comment:
The implicit tablespace is the parameter defaultTableSpace above in *translate* method

Reply:
Good catch. I will pass that to the method so can be used here.